distribution/ddf-common/src/main/resources/bin/setenv-wrapper.conf

```diff
@@ -51,15 +49,15 @@ wrapper.java.additional.26 = -XX:OnError=%KARAF_HOME%/bin/ddf_on_error.sh wrappe
 # security on your system. It should be used with caution, only to ascertain missing
 # permissions to be added to the default.policy file.
 #
-# wrapper.java.additional.30 = -Djava.security.manager=net.sourceforge.prograde.sm.PolicyFileGeneratorJSM
-# wrapper.java.additional.31 = -Dprograde.generated.policy=%KARAF_HOME%/generated.policy
-# wrapper.java.additional.32 = -Dprograde.use.own.policy=true
+# wrapper.java.additional.28 = -Djava.security.manager=net.sourceforge.prograde.sm.PolicyFileGeneratorJSM
+# wrapper.java.additional.29 = -Dprograde.generated.policy=%KARAF_HOME%/generated.policy
+# wrapper.java.additional.30 = -Dprograde.use.own.policy=true
 #
 # Production Security Permissions
-# wrapper.java.additional.30 = -Djava.security.manager=net.sourceforge.prograde.sm.ProGradeJSM
+# wrapper.java.additional.31 = -Djava.security.manager=net.sourceforge.prograde.sm.ProGradeJSM
```

Comment:
This should be 28

Reply:
I'm replacing all the commented out line numbers with `.<Number>` and a comment in the file to replace that value with the next available number